### PR TITLE
Added huggingface for keras siamese contrastive

### PR DESCRIPTION
**Pull request by rushic24**
As per the ongoing 🤗 Keras sprint, have updated the keras siamese contrastive notebooks for hugging face models

### CODE REVIEW

**Comment by rushic24** [comment]
I'm stuck with the autogen.py, It seems it is already blocked by something 
image

**Comment by rushic24** [comment]
How I generated :- 

```
black examples/vision/siamese_network.py          

cd scripts
python autogen.py add_example vision/siamese_network.py 
```

File: examples/vision/md/siamese_network.md [hunk 1]
```diff
@@ -83,10 +83,18 @@ positive_images_path = cache_dir / "right"
 
 <div class="k-default-codeblock">
 ```
-zsh:1: command not found: gdown
-zsh:1: command not found: gdown
-unzip:  cannot find or open left.zip, left.zip.zip or left.zip.ZIP.
-unzip:  cannot find or open right.zip, right.zip.zip or right.zip.ZIP.
+/mnt/hdd2/gender_detect/python38venv/lib/python3.8/site-packages/gdown/cli.py:127: FutureWarning: Option `--id` was deprecated in version 4.3.1 and will be removed in 5.0. You don't need to pass it anymore to use a file ID.
```
Comment (fchollet) [inline examples/vision/md/siamese_network.md:86]: Please manually remove log lines 86 to 93 -- they're just noise.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Review by fchollet — APPROVED** [review]
LGTM, thanks

**Comment by nashid** [comment]
@rushic24 

little bit confused as in the pull request I see: https://github.com/rushic24/keras-io/blob/e11524ac114b6997fe297a8f69ec342c016957db/examples/vision/siamese_network.py

**Comment by rushic24** [comment]
@nashid you're right, this should be in `examples/vision/siamese_contrastive.py` I am sorry, I'll fix it. 
@fchollet can you undo this merge

**Comment by nashid** [comment]
Thanks for your great work @rushic24 👍

**Comment by rushic24** [comment]
Updated here https://github.com/keras-team/keras-io/pull/941

**Comment by nashid** [comment]
@rushic24 great. Changes for the `triplet mining` with `siamese_network.py`  should be reverted though.

**Comment by rushic24** [comment]
I almost forgot, this is the revert of the original triplet mining changes.

**Pull request merged**